developer_guides/testbench/build_testbench.rst

```diff
@@ -0,0 +1,110 @@
+.. _build_testbench:
+
+Build Testbench and do first run
```

Comment:
We probably need a small introduction describing what Testbench is and what is the use case for Testbench. We could also have links in the other developer sections that point to it too.

Reply:
Yep, I'll add!

Reply:
I forgot to add valgrind run, it's great for memory violations search that I'll add it too.

Reply:
Done, I added a Introduction section to index.rst.